loops-and-arrays.js

```diff
@@ -12,9 +12,9 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
-
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i] + 'je')
+}
```

Comment:
je bent hier vergeten de namen in de array te vervangen. je print ze nu uit door "je" erachter te plakken. maar als je console.log(names) doet krijg je de namen weer zonder "je"